scene/main/canvas_item.cpp

```diff
@@ -292,15 +292,10 @@ void CanvasItemMaterial::_bind_methods() {
 
 CanvasItemMaterial::CanvasItemMaterial() :
 		element(this) {
-	blend_mode = BLEND_MODE_MIX;
-	light_mode = LIGHT_MODE_NORMAL;
-	particles_animation = false;
-
 	set_particles_anim_h_frames(1);
 	set_particles_anim_v_frames(1);
 	set_particles_anim_loop(false);
 
-	current_key.key = 0;
 	current_key.invalid_key = 1;
```

Comment:
How about this one?

Reply:
I wasn't sure how to initialize it, because it's inside some struct.
```
union MaterialKey {
		struct {
			uint32_t blend_mode : 4;
			uint32_t light_mode : 4;
			uint32_t particles_animation : 1;
			uint32_t invalid_key : 1;
		};
```
Trying to initialize it breaks compilation.